cpp/include/cudf/io/types.hpp

```diff
@@ -21,6 +21,7 @@
 
 #pragma once
 
+#include <cudf/table/table.hpp>
```

Comment:
this header was missing; found out when I added a mostly empty source file that includes `types.hpp`.